### ISSUE TITLE
How to output the found line or part of it in a custom rule?

### ISSUE THREAD

**Issue by Pe-te** [realm/SwiftLint#2003]
I have a custom rule that warns about comments with TODO in them:

```yaml
  custom_todo:
    include: ".*swift"
    name: "TODO"
    regex: "((?i)(TODO|FIXME))"
    match_kinds:
      - comment
    message: "Unfinished Code!"
    severity: warning
```

With multiple TODOs in the code, I only see the same short warning message for all, so I wanted to also show the text that triggered it. Is there any way to achieve this?

I was expecting to use something like this, but it doesn't work:

```yaml
regex: "TODO(.*)"
message: "Unfinished code! \1"
```

In case that regex matches are not possible, can I at least add the line that triggered it?

**Comment by marcelofabri** [comment]
This is not current supported, but I think it would be a good addition.
